services/minespace-web/src/styles/overrides/filepond-overrides.scss

```diff
@@ -5,7 +5,7 @@
 
 /* the text color of the drop label*/
 .filepond--drop-label {
-  color: $gov-grey-dark; //#555;
+  color: rgba(49, 49, 50, 1); //#555;
```

Comment:
I think this is the same as the variable $font-colour.

Reply:
Yep, you are correct. I corrected this with $font-colour.